### ISSUE TITLE
Collapse the headings in Table of Contents by default

### ISSUE THREAD

**Issue by user1823** [sumatrapdfreader/sumatrapdf#3957]
For some files, the headings in the Table of Contents are collapsed by default. For some, they are not.

Please make it so that the headings are collapsed in all files by default. Otherwise, it becomes very difficult to find the correct entry when there are 1000s of entries there.

Also, some files contain several levels of headings and sub-headings. In that case, expanding a heading should not cause all of its subheadings to be expanded too.

**Comment by GitHubRulesOK** [comment]
The initial state of a TOC (bookmarks section) is usually set by the Document (PDF) once you switch remember on then the status tracks your preferences so the best way to set them off is in a PDF editor or via a command line PDF rewriter, since SumatraPDF is not able to edit PDF core that is way beyond its lightweight viewer capabilities.

related to https://github.com/sumatrapdfreader/sumatrapdf/issues/1998 and https://github.com/sumatrapdfreader/sumatrapdf/issues/718#issuecomment-418761410 

Default set to open Bookkmarks Sidebar without remember document state thus will be as per Author Controls
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/e36ac97d-1322-4cdc-b059-4697b056dd10)

Open a document with multiple levels of Authors bookmarks they are ALL collapsed
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/d72852bd-c917-496b-be32-db12211d54ae)

Use the expand & collapse control
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/8e9173b3-dfe8-4065-a23b-756c9d9f2bc3)
Now they are ALL expanded
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/fb9cc2bf-9056-4e87-9c26-0b3de11a5b67)


Basically there is no issue With SumatraPDF it is an issue with individual PDF and any nested settings they request.

**Comment by kjk** [comment]
We respect the information stored in PDF file. If we ignore that, someone will open a bug asking to not ignore it.

You can right-click on table of contents tree and use `Collapse all`

**Comment by user1823** [comment]
@kjk, I respect your decision.

But, most of the PDF readers that I use ignore this information.

Probably, the best solution is that SumatraPDF respects this information by default but ignores it when configured in the Advanced Settings.

This way, SumatraPDF can provide an experience similar to other PDF readers.
